src/ol_infrastructure/lib/consul.py

```diff
@@ -1,12 +1,17 @@
 import base64
 import json
-import textwrap
+
 import yaml
 
 
 def build_consul_userdata(env_name):
-    """ Builds consul userdata for cloud init, to create auto scale group and launch configs
-        TODO setup for consul_esm """
+    """Build consul userdata for cloud init, to create auto scale group and launch configs.
+
+    :param env_name: consul environment being built
+    :returns: base64-encoded yaml cloud init data
+
+    TODO setup for consul_esm
```

Comment:
There shouldn't actually need to be anything done in cloud-init for consul-esm. It should be able to be entirely naive to the environment that it's running in.